docs/scenarios/jdk_8_to_17_migration/README.md

```diff
@@ -326,8 +356,8 @@ Konveyor analysis detected two dependency-related issues in `pom.xml`:
 #### Requesting a Fix
 
 We will solve both the issues together this time. Clicking on the **tool icon** next to the file name initiates the resolution process for both the issues.
-![pom_issues.png](images/pom_issues.png)
-![pom_fixes.png](images/pom_fixes.png)
+![pom_files_fixes.png](images/pom_files_fixes.png)
+![pom_results.png](images/pom_results.png)
```

Comment:
I see a mix of diff view amd merge view solutions. 

can you pick either the merge view or doff view and use that throughout the solution fixes? it helps avoiding confusion for new users.